### DISCUSSION TITLE
Citus replication model

### DISCUSSION THREAD

**Discussion by huseynsnmz** [citusdata/citus#4597] [Q&A] [answered]
Hello,

I saw "citus.replication_model" parameter in issue pages and I couldn't find any information in documents. I find description in PostgreSQL settings but I couldn't understand what it is used for. What is the difference between citus.replication_model = 'statement and citus.replication_model = 'streaming' ?

**Comment by agedemenli** [comment, marked as answer]
Difference in functionality: [Citus Blog](https://www.citusdata.com/blog/2016/12/15/citus-replication-model-today-and-tomorrow/) (very useful, recommended)

Difference as restrictions: `streaming` mode can only be used when `citus.shard_replication_factor = 1` and only if you use hash distribution.  And it's not allowed for deprecated `master_create_distributed_table`. While `statement` is allowed with greater replication factors and other distribution methods.

see `create_distributed_table.c`

**Comment by huseynsnmz** [comment]
Oh, I understand. Thank you. :)
